'Formatted  table heading and added dasboard nav bar'

IsaackMwenda opened the pull request:
Separated table heading and body by formatting the colours on both.
I also added a navigation bar and wrapped the table in a container element.
<img width="1277" alt="dashboard" src="https://user-images.githubusercontent.com/23135322/49020142-c4959980-f1a0-11e8-80b1-8b35bdef4caf.png">

IsaackMwenda committed:
'Formatted  table heading and added dasboard nav bar'

Separated table heading and formated the colours to separate th and table body.
Added a nav bar

lukechurch commented:
Thanks @IsaackMwenda, per discussion I'll let Rita review first.

For things like this, it's pretty difficult to imagine what it looks like - could you also take a quick screenshot and embed it in the PR comment?

IsaackMwenda commented:
> Thanks @IsaackMwenda, per discussion I'll let Rita review first.
> 
> For things like this, it's pretty difficult to imagine what it looks like - could you also take a quick screenshot and embed it in the PR comment?
@lukechurch Great have added a screenshot.

lukechurch commented on public/coda/coding_progress.html:
```diff
@@ -27,28 +28,34 @@
-    
+<nav class='nav'>
+    <div class="nav-wrapper">
+        <a href="https://www.africasvoices.org/wp-content/themes/africa/static/img/africas-voices-logo-pad.svg" class="brand-logo center">Operations Dashboard</a>
```
This looks a bit odd. Could you elaborate on what you were aiming to achieve with the <a> tag referencing an image as its target?

IsaackMwenda replied:
I wanted to include the AVF logo with the dashboard title. Have fixed it 
![image](https://user-images.githubusercontent.com/23135322/49067066-028fcd80-f234-11e8-8bf7-4336e6516898.png)

lukechurch commented on public/coda/coding_progress.html:
```diff
@@ -27,28 +28,34 @@
-
-<table>
+<div class="container">
+<table class='datatable'>
```
Please could you indent this as it's enclosed within the div

IsaackMwenda replied:
Noted

lukechurch commented on public/coda/coding_progress.html:
```diff
@@ -27,28 +28,34 @@
-        <td>Messages</td>
-        <td>Messages with a label</td>
-        <td>Done</td>
+        <th>Dataset</th>
```
Nice :)

lukechurch commented on public/coda/coding_progress.js:
```diff
@@ -4,10 +4,10 @@ progress_fs.settings(settings);
 function update_progress_ui(data) {
     console.log("update_ui: " + JSON.stringify(data));
     var status_body = document.getElementById('coding_status_body');
+
```
I don't think there's a need for this additional whitespace? it doesn't really help reading and results in less code on the screen

IsaackMwenda replied:
Great have removed it

lukechurch commented on public/coda/styles.css:
```diff
@@ -0,0 +1,29 @@
+    background-color: #5CA5AE;
+    color: white;
+    height: 50 px;
+}
```
New line at the end of the file please, so that if you 'cat' the code into a terminal, it ends nicely

lukechurch reviewed (commented):
A few tweaks to go before we land this please, nothing major though and a good step in the right direction visually

lukechurch commented:
Hi @IsaackMwenda - after you've made the changes locally, you need to commit them to your local copy of the branch, and then push that branch to Github. That way I'll be able to see the changes you've made, and review again

IsaackMwenda committed:
'Fixed identation,whitespaces and added new line in each file'

Formatted identation
Removed whitespaces
Added new line at the end of each file

IsaackMwenda committed:
Delete settings.json

ritazagoni commented on public/coda/styles.css:
```diff
@@ -0,0 +1,18 @@
+    max-width:initial;
+    height:80px;
+  }
+  .nav-link {
```
Looks like this should have the same indentation as the others as on the same level?

IsaackMwenda replied:
Noted. Thank you

ritazagoni reviewed (approved):
Thanks @IsaackMwenda one minor comment. Once that's fixed and pushed pls request final review from Luke.

IsaackMwenda committed:
Fixed identation

Fixed identation

IsaackMwenda committed:
Merge branch 'dashboard-development' of https://github.com/AfricasVoices/Operations-Dashboard into dashboard-development

IsaackMwenda commented:
@lukechurch Have fixed indentation, removed white spaces and added a new line at the end of each file. Kindly review.

lukechurch reviewed (approved):
Looks good to me! Thanks @IsaackMwenda 

Please merge and push to firebase hosting
